architecture/service_catalog/index.adoc

```diff
@@ -0,0 +1,191 @@
+[[architecture-additional-concepts-service-catalog]]
+= Service Catalog
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[service-catalog-overview]]
+== Overview
+
+[NOTE]
+====
+The service catalog is currently a Technology Preview feature.
+ifdef::openshift-origin,openshift-enterprise[]
+To opt-in during installation or upgrade, see Installation & Configuration (TODO link).
+endif::[]
+====
+
+When developing microservices-based applications to run on cloud native
+platforms, there are many ways to provision different resources and share their
+coordinates, credentials, and configuration, depending on the service
+provider and the platform.
+
+To give developers a more seamless experience, {product-title} includes a
+_service catalog_, an implementation of the
+link:https://openservicebrokerapi.org/[Open Service Broker API] (OSB API) for
+Kubernetes. This allows users to connect any of their applications deployed in
+{product-title} to a wide variety of service brokers.
+
+The service catalog allows cluster administrators to integrate multiple
+platforms using a single API specification. The {product-title} web console
+aggregates and lists the contents of configured service brokers, allowing users
```

Comment:
You might want to avoid the term 'aggregates' as 'the aggregator' is a related component to service catalog.  It might be sufficient to say 'The web console shows the service classes offered by the brokers in the catalog'